### PR TITLE
api change in package.js for Meteor 1.2

### PR DESCRIPTION
**Pull request by sebastianconcept**
This allows it to load in Meteor 1.2

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
     name: "chadkruser:meteor-accounts-stripe-connect",
-    version: "0.0.6",
+    version: "0.0.7",
     git: "https://github.com/chadokruse/meteor-accounts-stripe-connect.git",
 });
 
-Package.on_use(function(api) {
-    api.versionsFrom('METEOR@1.0');
+Package.onUse( function (api) {
+    api.versionsFrom('1.2');
```
Comment (benjick) [inline package.js:9]: Is this really necessary? `onUse` and `addFiles` should work in 1.0 as well

File: package.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
 Package.describe({
     summary: "Login service for Stripe accounts using Stripe Connect",
     name: "chadkruser:meteor-accounts-stripe-connect",
-    version: "0.0.6",
+    version: "0.0.7",
```
Comment (benjick) [inline package.js:4]: Should be 1.0.0 if you break backwards compatibility

File: package.js (hunk 1 above)
Comment (sebastianconcept) [inline package.js:9]: I've intended to make it work on 1.2

I'm not sure how to proceed about 1.0, what do you suggest?

File: package.js (hunk 2 above)
Comment (sebastianconcept) [inline package.js:4]: good catch

**Pull request merged**